src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -338,7 +346,7 @@ class instance_naxxramas : public InstanceMapScript
                     _kelthuzadgateGUID = pGo->GetGUID();
                     if (GetBossState(BOSS_SAPPHIRON) == DONE && _speakTimer==0)
                         pGo->SetGoState(GO_STATE_ACTIVE);
-               	   break;
+               	    break;
```

Comment:
space

Reply:
This space is intended. While I was correcting my fault indentation I also checked the indentation of the cases above and saw that this "break" statement is missing a space in front of it.

Here is a screenshot of the original source in the current rev:

![indentation](https://user-images.githubusercontent.com/23292416/77860321-dd87e200-720e-11ea-829c-a56aad9f9a68.png)

Thought I could fix it within this PR, but I can also remove it if you're saying that I'm wrong tho :)

Reply:
Oh oh, then it's good :)